src/bootstrap/llvm.rs

```diff
@@ -760,7 +762,9 @@ fn configure_cmake(
             if target.contains("apple") || target.contains("windows") {
                 ldflags.push_all("-static-libstdc++");
             } else {
-                ldflags.push_all("-Wl,-Bsymbolic -static-libstdc++");
+                ldflags.push_all(
+                    "-Wl,-Bsymbolic-functions -fno-semantic-interposition -static-libstdc++",
```

Comment:
`-Bsymbolic-functions` is weaker (it only applies to functions) than the existing `-Bsymbolic`. You should keep `-Bsymbolic` unless it causes issues with `-fno-semantic-interposition`

Reply:
Uh, results not nice, reverted back :-(